nogo_config.json

```diff
@@ -139,5 +139,34 @@
       "vendor/": "vendor doesn't pass vet",
       "external/": "externaldoesn't pass vet"
     }
+  },
+  "errcheck": {
+    "exclude_files": {
```

Comment:
not sure if the `include_files` directive would have been easier for you here, but works for me.